### PR TITLE
enable to set ip address via cmdline

### PR DESCRIPTION
**Pull request by Saza-ku**
We can now set IP address in this way.

```
qemu-system-x86_64 \
    ... \
    -append "ip=10.0.2.15 gateway=10.0.2.2"
```

### CODE REVIEW

File: src/param.zig [hunk 1]
```diff
@@ -5,6 +5,7 @@ const Ip4Address = std.os.linux.sockaddr;
 const Params = struct {
     addr: u32 = undefined,
     subnetmask: u32 = undefined,
+    gateway: u32 = undefined,
```
Comment (ainozaki) [inline src/param.zig:8]: Should we set default value rather than `undefined` ?
Or insert assertion before calling lwip?

Comment (Saza-ku) [reply src/param.zig:8]: How about make it nullable and the default value null?

Comment (ainozaki) [reply src/param.zig:8]: Looks good!

**Comment by Saza-ku** [comment]
@ainozaki Fixed!

**Review by ainozaki — APPROVED** [review]
Thanks!

**Pull request merged**